changelog/db.changelog-1.42.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-3.9.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd">
+  <changeSet author="John Watson" id="1">
+    <tagDatabase tag="v1.42.0" />
+  </changeSet>
+  <changeSet author="John Watson" id="2">
+    <insert tableName="holdCategories">
+      <column name="holdCategoryId" value="116" />
+      <column name="schemeId" value="6" />
+      <column name="name" value="Manual ledger hold" />
+    </insert>
+    <insert tableName="holdCategories">
+      <column name="holdCategoryId" value="117" />
+      <column name="schemeId" value="7" />
+      <column name="name" value="Manual ledger hold" />
```

Comment:
Where these hold categories missing for these schemes?

Reply:
Yeah we missed them earlier.